rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMStatsView.java

```diff
@@ -27,6 +27,7 @@
 
 import java.util.Date;
 import java.util.LinkedHashMap;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)